sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BuiltInDataSourceWriteBenchmark.scala

```diff
@@ -53,7 +56,16 @@ object BuiltInDataSourceWriteBenchmark extends DataSourceWriteBenchmark {
 
     formats.foreach { format =>
       runBenchmark(s"$format writer benchmark") {
-        runDataSourceBenchmark(format)
+        if (format.equals("Parquet")) {
+          ParquetProperties.WriterVersion.values().zip(Seq("V1", "V2")).foreach {
```

Comment:
In general, we had better avoid magic string. Please use `ParquetProperties.WriterVersion.PARQUET_1_0.toString` and `ParquetProperties.WriterVersion.PARQUET_2_0.toString`.

Reply:
ok, I will fix this later

Reply:
5155fd7 fix this, let me update the bench-results

Reply:
@dongjoon-hyun done